lib/wget.js

```diff
@@ -89,7 +89,7 @@ function download(src, output, options, _parentEvent, redirects) {
             // Data handlers
             res.on('data', function(chunk) {
                 downloadedSize += chunk.length;
-                downloader.emit('progress', downloadedSize/fileSize);
+                downloader.emit('progress', downloadedSize);
```

Comment:
This will break many users implementations, I have added `downloader.emit('bytes', downloadedSize);` to accomplish this feature.